scripts/feed/PostInteraction.js

```diff
@@ -3,8 +3,32 @@ handles the rendering and interaction of a post element
 */
 
 // import deletePost, favoritePost from ./data/TransientState.js
+import { deletePost, favoritePost } from '../data/TransientState.js';
+
 // attach 2 click event listeners -- both outside any function:
     // Delete button
     // Favorite button
-// declare PostInteraction function
-// export PostInteraction function to PostList
+document.addEventListener('click', handleDeleteButtonClick);
+document.addEventListener('click', handleFavoriteButtonClick);
+
+// declare and export PostInteraction function
+export const PostInteraction = (postId) => {
+  const handleDeleteButtonClick = (event) => {
```

Comment:
Maybe we could define these callback functions outside the scope of this function?